src/state/process.rs

```diff
@@ -449,12 +449,7 @@ where
     /// interval. Since it is possible to have received and cached the current height's proposals,
     /// votes and quorum certificates before, these should be re-checked as goto new height.
     /// Finally, trigger SMR to goto new height.
-    async fn goto_new_height(
-        &mut self,
-        ctx: Context,
-        status: Status,
-        get_last_flag: bool,
```

Comment:
Why remove this flag?

Reply:
Overlord will not handle messages that below self height, so that the authority list of the last height is useless.